src/lib/link.spec.ts

```diff
@@ -4,12 +4,20 @@ import test from 'ava';
 import { State } from '../codec/ibc/core/channel/v1/channel';
 
 import { Link } from './link';
-import { ics20, randomAddress, setup, simapp, wasmd } from './testutils.spec';
+import {
+  ics20,
+  randomAddress,
+  setup,
+  simapp,
+  TestLogger,
+  wasmd,
+} from './testutils.spec';
 
 test.serial('establish new client-connection', async (t) => {
+  const logger = new TestLogger();
```

Comment:
So, if I want detailed logs from this method, I would just use `const logger = new TestLogger(true)` ?

Reply:
Yes, passing `true` will turn on console logging for all levels. Does it need to be configurable? I was imagining that if we wanted to see what was being logged while running tests it would be temporary and we'd want to see everything.

Reply:
That sounds good. And we can enable it on a per-test basis.

Seems like a good design, I just wanted to ensure I understood